### DISCUSSION TITLE
Manual / human-assisted captcha solving

### DISCUSSION THREAD

**Discussion by tallridge** [FlareSolverr/FlareSolverr#1188] [Q&A] [answered]
An option will tell FlareSolverr to spawn browser window (just like HEADLESS=false) without any solving machinery, providing ability to solve captcha by hand and then respond with cookie as usual.

# Why need this
I'm using FlareSolverr with a few applications and very low RPS (about once a week). FlareSolverr is just so convenient that it helps to dodge some non-automatable actions like:
1. Install weird browser extensions to save cookies in weird format
2. Think where to place those savings and how to reformat them.
3. Maintain constant IP between (1) browser and cookie user.

With FlareSolverr I wrote simple scripts and it works... Except now it doesn't because of well-known issue. If that issue about automated solver only, maybe we can optionally leave captcha solving to human.

**Comment by ilike2burnthing** [comment, marked as answer]
Why not use `headless=false`?

**Reply by tallridge** [reply]
You are right, sorry for creating this.
